packages/cli/src/index.ts

```diff
@@ -20,58 +18,75 @@ const hintStyle = chalk.hex('#6a5f88');
 
 // Globally configurable arguments
 const globalConfig: { [name: string]: Options } = {
-  'gl-token': {
+  'gb-token': {
+    alias: 'gl-token',
     desc: 'Your Gitlab Personal Token',
     type: 'string',
-    defaultValue: process.env.GITLAB_TOKEN,
+    defaultValue: process.env.GITBEAKER_TOKEN || process.env.GITLAB_TOKEN,
```

Comment:
First big uff.
I think this is going beyond the branding. This clearly is a GITLAB_TOKEN and this is something gitlab ci will even expose.

Reply:
Yea there is gonna be alot of clean ups on the way as I move it all to the monorepo. That's why I'm stressing the test coverage. The change had it previously , i just added the gitbeaker part. You think keeping the GITLAB prefix might cause some conflicts down the road?

Reply:
I must admit I missed the `||` and only saw `GITBEAKER_TOKEN`.

Fine as long as `GITLAB_TOKEN` stays. I see no conflicts I was worried that we only had `GITBEAKER_TOKEN` which would IMHO break a lot of things.

Reply:
I agree, thats why i kept it :D Should all still work fine